spartan/aztec-network/files/config/setup-service-addresses.sh

```diff
@@ -107,6 +107,15 @@ else
   BOOT_NODE_ADDR="http://${SERVICE_NAME}-boot-node.${NAMESPACE}:${BOOT_NODE_PORT}"
 fi
 
+# Configure Full Node address
+if [ "${EXTERNAL_FULL_NODE_HOST}" != "" ]; then
```

Comment:
Seems like we are not setting this variable anywhere? Should we add it to the `_helpers` next to the other `EXTERNAL-*` vars?

Reply:
Yes, thanks. Hate this K8s config. It's all over the place.